modules/file-system/parallelstore/scripts/install-daos-client.sh

```diff
@@ -68,6 +68,11 @@ sed -i "s/#.*transport_config/transport_config/g" $daos_config
 sed -i "s/#.*allow_insecure:.*false/  allow_insecure: true/g" $daos_config
 sed -i "s/.*access_points.*/access_points: $access_points/g" $daos_config
 
+# Move agent log destination from /tmp/ (default) to /var/log/daos_agent/
+mkdir /var/log/daos_agent
```

Comment:
I suggest `mkdir -p` here. It would be _extremely_ strange for a linux system to exist without `/var/log` but the `-p` makes this command fail-proof.